resources/js/Components/LeftMenu.tsx

```diff
@@ -1,11 +1,70 @@
+import axios from "axios";
 import Brand from "./Brand";
 import {
     HomeIcon,
     ArchiveBoxIcon,
     BookOpenIcon,
 } from "@heroicons/react/24/solid";
+import useSWR from "swr";
+
+interface CategoryParameters {
```

Comment:
I would move these types to the resources/js/common.ts file and rename them to just Category and CategoryLink.